src/components/BlogsContainer.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="mt-12 max-w-lg mx-auto grid gap-5 lg:grid-cols-3 lg:max-w-none"><slot>No posts are available at this moment.</slot></div>
```

Comment:
There is no need to create a new component for this line, just create a v-if v-else condition

Reply:
Should I entirely remove this component?